arlpy/uwa.py

```diff
@@ -223,3 +223,37 @@ def bubble_soundspeed(void_fraction, c=soundspeed(), c_gas=340, relative_density
     """
     m = _np.sqrt(relative_density)
     return 1/(1/c*_np.sqrt((void_fraction*(c/c_gas)**2*m+(1-void_fraction)/m)*(void_fraction/m+(1-void_fraction)*m)))
+
+def pressure(x, sensitivity, gain, volt_params=None):
```

Comment:
I am trying to understand the use case to make sure a user knows that this is to be used for. Perhaps renaming the function once we clarify this might be helpful. The function name `pressure()` suggests to me that it converts something to pressure? What does it convert? Volts to pressure? ADC units to pressure?

Reply:
```uwa.pressure()``` converts a real signal either in volts or in ADC units (bit depth) to pressure. If the input argument ```volt_params``` is provided, the input is in ADC units, else is in volts. 

Agree that the function could be better named.

Reply:
Agree with the function, but I have trouble understanding the documentation, especially `volt_params`. Why is it a tuple (or is it meant to be a tuple), when other parameters are arguments? Also, it might help to provide some examples (like in many other functions in `arlpy`) to help the user.

Reply:
Number of bits (`nbits`) and reference voltage (`V_ref`) are required to convert bit depths to voltages. It would be more compact if these parameters can be grouped together as one input argument, and hence `volt_params` is a tuple/list. 

To avoid any confusion, I was thinking shall we remove the `volt_params` so that the function `pressure` only accepts `x` as a real signal in voltage? 

Test cases and examples have been added.